content/storage-history/storage.textile

```diff
@@ -0,0 +1,109 @@
+</a>
+
+Read more about the "History API":/storage-history/history or the "Realtime connection state recovery":/realtime/connection/#connection-state-recovery.
+
+
+h2(#all-message-persistence). Persist all messages - 24 or 72 hours
+
+If you need to retain messages for longer than the default two minutes you can enable "persisted history" when setting a "channel rule":/general/channel-rules-namespaces. When "persisted history is enabled":#persisted-history for a channel any messages will be stored on disk for 24 for free tiers or 72 hours for paid tiers.
+
+In this scenario, the History API can be used to retrieve the persisted messages. This is illustrated in the following diagram:
+
+<a href="/images/diagrams/history-persist-all-messages.png" target="_blank">
+  <img src="/images/diagrams/history-persist-all-messages.png" style="width: 100%" alt="Persist All Messages">
+</a>
+
+Every message that is persisted to, or retrieved from, disk counts as an extra message towards your monthly quota. For example, with persistence enabled a published message counts as two messages for your monthly quota. If the message is then retrieved another message will be deducted from your monthly quota.
+
+To enable history on a channel add a @persist all messages@ "channel rule":/general/channel-rules-namespaces in the settings of your Ably "dashboard":https://ably.com/dashboard.
+
+Foldable box
+
+    Select the settings tab from your Ably "dashboard":https://ably.com/dashboard :
+
+    <a href="/images/screenshots/storage/dashboard-settings.png" target="_blank">
+      <img src="/images/screenshots/storage/dashboard-settings.png" style="width: 100%" alt="Settings button">
+    </a>
+
+    Scroll down and click "Add new rule":
+
+    <a href="/images/screenshots/storage/dashboard-channel-rule.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-channel-rule.png" style="width: 100%" alt="Add new rule button">
+    </a>
+
+    Tick "Persist all messages":
+
+    <a href="/images/screenshots/storage/dashboard-persist-all.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-persist-all.png" style="width: 100%" alt="Persist all messages checkbox">
+    </a>
+
+    Click "Create channel rule":
+
+    <a href="/images/screenshots/storage/dashboard-save.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-save.png" style="width: 100%" alt="Create channel rule button">
+    </a>
+
+
+h2(#persist-last-message). Persist last message - 365 days
+
+You can also choose to persist just "the last message":/general/channel-rules-namespaces on a channel, which persists the last published message for one year. 
+
+This message is not retrieved using the History API, you must use the "channel rewind":/channels/options/rewind mechanism, attached to a channel with @rewind=1@. Note that only regular channel messages can be stored, not presence messages. 
```

Comment:
"rewind mechanism" and "rewind option" used interchangeably on this page. I'd opt for "rewind channel option"